npc/re/merchants/enchan_verus.txt

```diff
@@ -1129,15 +1130,15 @@ verus04,163,219,4	script	PLUTO_09#pa0829	4_SCR_AT_ROBOTS,{
 				.@fail = true;
 				.@reac_rnd2 = rand(1,13000);
 				if (.@reac_rnd2 < 2)
-					.@en_name = 969;	// Gold
+					.@en_name = 6962,5;	// Current: 5 OldTank. Original: Gold // .@en_name = 969;
 				else if (.@reac_rnd2 < 1002)
-					.@en_name = 999;	// Steel
+					.@en_name = 6962,5;	// Current: 5 OldTank. Original: Steel // .@en_name = 999;
 				else if (.@reac_rnd2 < 3002)
-					.@en_name = 998;	// Iron
+					.@en_name = 6962,5;	// Current: 5 OldTank. Original: Iron // .@en_name = 998;
 				else if (.@reac_rnd2 < 8002)
-					.@en_name = 1002;	// Iron_Ore
+					.@en_name = 6962,5;	// Current: 5 OldTank. Original: Iron_Ore // .@en_name = 1002;
 				else
-					.@en_name = 7054;	// Brigan
+					.@en_name = 6962,5;	// Current: 5 OldTank. Original: Brigan // .@en_name = 7054;
```

Comment:
To refund the old fuel tanks we can actually just remove the logic here from lines [1131-1141](https://github.com/Latiosu/chocobotro/pull/12/files#diff-8933c2cb2f0ccaa501e5e6af9559e785c896bf14a3ea0f4e8a9fb7b9f4ba0a80R1131-R1141) and then move lines [1147-1148](https://github.com/Latiosu/chocobotro/pull/12/files#diff-8933c2cb2f0ccaa501e5e6af9559e785c896bf14a3ea0f4e8a9fb7b9f4ba0a80R1147-R1148) into [1152](https://github.com/Latiosu/chocobotro/pull/12/files#diff-8933c2cb2f0ccaa501e5e6af9559e785c896bf14a3ea0f4e8a9fb7b9f4ba0a80R1152) (for success), but we'll need to add curly braces since the current syntax only handles a single statement within the if/else.

Reply:
So lines 1145-1154 would then look something like:
```
else {
    specialeffect EF_SPELLBREAKER;
    mes "[PLUTO_09]";
    if (.@fail)
	    mes "What a pity, the design failed.";
    else {
            delitem 6962,5;// OldTank
            getitem .@en_name,1;
	    mes "Great, the design was made.";
    }
}
```